src/core/updaters.go

```diff
@@ -0,0 +1,259 @@
+package core
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+	"strconv"
+	"time"
+
+	"github.com/trustwallet/assets-go-libs/pkg"
+	"github.com/trustwallet/assets-go-libs/pkg/asset"
+	"github.com/trustwallet/assets-go-libs/pkg/validation/info"
+	"github.com/trustwallet/assets-go-libs/src/client/binance/dex"
+	"github.com/trustwallet/assets-go-libs/src/client/binance/explorer"
+	"github.com/trustwallet/assets-go-libs/src/config"
+	assetlib "github.com/trustwallet/go-primitives/asset"
+	"github.com/trustwallet/go-primitives/coin"
+	"github.com/trustwallet/go-primitives/numbers"
+	"github.com/trustwallet/go-primitives/types"
+)
+
+const (
+	assetsPage       = 1
+	assetsRows       = 1000
+	marketPairsLimit = 1000
+	tokensListLimit  = 10000
+
+	twLogoURL       = "https://trustwallet.com/assets/images/favicon.png"
```

Comment:
why do we need `twLogoURL` ?

Reply:
i assume it's used somewhere in the app? Is tokenlist.json used by the app? if it's just some info - I can remove it